src/ripple/app/misc/Validations.cpp

```diff
@@ -346,9 +347,12 @@ class ValidationsImp : public Validations
     }
 
     LedgerToValidationCounter getCurrentValidations (
-        uint256 currentLedger, uint256 priorLedger) override
+        uint256 currentLedger,
+        uint256 priorLedger,
+        LedgerIndex cutoffBefore) override
     {
         auto const cutoff = app_.timeKeeper().now().time_since_epoch().count() - LEDGER_VAL_INTERVAL;
+        auto const lCutoff = cutoff + LEDGER_VAL_INTERVAL - LEDGER_VAL_LOCAL;
```

Comment:
Agreed. Subtracting out `LEDGER_VAL_INTERVAL` only to add it back is confusing, wasteful, and is going to be fragile in the future.